metals/src/main/scala/scala/meta/metals/Main.scala

```diff
@@ -48,9 +48,11 @@ object Main {
     val tracePrinter = Trace.setupTracePrinter("LSP")
     val exec = Executors.newCachedThreadPool()
     val ec = ExecutionContext.fromExecutorService(exec)
+    val sh = Executors.newSingleThreadScheduledExecutor()
     val initialConfig = MetalsServerConfig.default
     val server = new MetalsLanguageServer(
-      ec,
+      ec = ec,
+      sh = sh,
```

Comment:
Ok, this isn't necessary but I thought that maybe it's easier to accept `sh` rather than rely on default parameter